examples/BERT/data.py

```diff
@@ -138,3 +158,29 @@ def find_language_id(self, file_name):
                  'tl_XX', 'tn_BW', 'tr_TR', 'ug_CN', 'uk_UA',
                  'ur_PK', 'ur_PK_rom', 'uz_UZ', 'vi_VN', 'wo_SN',
                  'xh_ZA', 'yi_DE', 'yo_NG', 'zh_CN', 'zh_TW']
+
+CC100_FILES_LINES = {
```

Comment:
This could be great raw dataset because it likely benefits from streaming a lot.